kyma-project/test-infra
# Test eventing upgrade job with prod profile

**Description**

Changes proposed in this pull request:

- use prod profile to test NATS JetStream in clustering mode
- use better machine to test the Kyma eventing with prod profile
- check the green runs [here](https://status.build.kyma-project.io/?job=vladislavpaskar_test_of_prowjob_pre-main-kyma-gardener-gcp-even)
